### PR TITLE
CMDCT-3292: Update Home Health SPA IDs

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Updating SPA IDs based on BO feedback.

### Related ticket(s)

CMDCT-3292

---
### How to test

N/A

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/libs/spaLib.ts [hunk 1]
```diff
@@ -399,7 +399,7 @@ export const SPA: { [year: string]: SPAi[] } = {
       name: "Chronic Care Management for Individuals with Serious and Persistent Mental Health Conditions",
     },
     { id: "18-0007", state: "DC", name: "Chronic Conditions" },
-    { id: "22-0003", state: "IA", name: "Iowa Health Home Services" },
```
Comment (BearHanded) [inline services/ui-src/src/libs/spaLib.ts:402]: Are these things that should be changing globally or year over year? I'm assuming these changes modify every year?

Comment (karla-vm) [reply services/ui-src/src/libs/spaLib.ts:402]: No, these ID changes were requested for 2024 specifically, we don't need to retroactively update them.

Comment (BearHanded) [reply services/ui-src/src/libs/spaLib.ts:402]: I misunderstood what was going on in the file, my bad. I missed this was a 2024 entry.

**Review by BearHanded — APPROVED** [review]

**Review by ailZhou — APPROVED** [review]
LGTM!

**Pull request merged**